chia/consensus/default_constants.py

```diff
@@ -86,6 +88,7 @@
     PLOT_FILTER_64_HEIGHT=uint32(15592000),
     # June 2033
     PLOT_FILTER_32_HEIGHT=uint32(20643000),
+    PLOT_DIFFICULTY_INITIAL=uint8(2),
```

Comment:
just to have said it, i'm a bit iffy about some v2 items being suffixed and others not.  i understand that in this case there is no v1 of the same name.  feel free to close this after reading.